packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,507 @@
+];
+
+<CardView items={items} height="500px" maxWidth="800px" layout={GridLayout} aria-label="Dynamic CardView Pokemon example">
+  {item => (
+    <Card textValue={item.name} width={item.width} height={item.height}>
+      <Image src={item.src} />
+      <Heading>{item.name}</Heading>
+      <Text slot="detail">PNG</Text>
+      <Content>{item.type}</Content>
+    </Card>
+  )}
+</CardView>
+```
+
+### Cards
+A standard Card consists of a preview image and a title, and allows for a optional detail area next to the title and an optional content area immediately below the
+title. The image, title, and content area can be populated by providing a [Image](Image.html), [Heading](Heading.html), and [Content](Content.html) respectively as children to
+the Card. The detail area can be populated through a [Text](Text.html) element with an accompanying `slot="detail"` prop. Please note that a Spectrum compliant Card should not contain
+focusable or interactive content within the content area and that the content itself should only display the most essential pieces of information. If you need to perform actions
+on a selected card(s), please use the ActionBar component in lieu of adding an action menu to the Card itself. See the [Card Actions](#card-actions) section for an example.
+
+The properties that the Card supports can be found in the [Props](#props) table below. Note that the `height` and `width` props refer to the image's height and width, not the Card's height and
+width. These values are used to calculate the proper positioning and size of each card within the CardView's container.
+
+### Internationalization
+To internationalize a CardView, all text content within the Cards should be localized. The `aria-label` provided to the CardView should also be localized any.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of CardView is automatically flipped.
+
+## Layout
+CardView supports three different visual layouts: <TypeLink links={docs.links} type={docs.exports.GridLayout} />, <TypeLink links={docs.links} type={docs.exports.GalleryLayout} />
+and <TypeLink links={docs.links} type={docs.exports.WaterfallLayout} />. You can set the CardView's layout by passing the imported class directly to the `layout` prop as shown in the previous examples.
+This will handle instantiating said layout with its default options. Alternatively, you can modify the layout's various options (e.g. `margin`, `collator`, etc) by constructing your own instance of the
+class and providing that to the CardView instead. See the sections below for examples of this.
+
+### GridLayout
+The GridLayout renders each card with a uniform height and width regardless of the image size. The customizable options available to the GridLayout are as follows:
+
+// TODO figure out how to show a table for the GridLayoutOptions
```

Comment:
Still figuring out how to show these layout options, getting errors when trying to do a TypeLink for it. Perhaps the CardLayout types should be changed [here](https://github.com/adobe/react-spectrum/blob/main/packages/@react-types/card/src/index.d.ts#L29-L51) to be more specific for GridLayout/GalleryLayout/WaterfallLayout